spec/features/auth_spec.rb

```diff
@@ -0,0 +1,136 @@
+RSpec.feature 'User authentication' do
+  context 'With valid details' do
+    scenario 'sign up' do
```

Comment:
`sign up` is a feature - and we prefer to describe a happy path inside a scenario description string. E.g. here you could describe it like that:
`scenario 'user signs up'`